org.metaborg.meta.lang.template/metaborg.yaml

```diff
@@ -25,9 +25,17 @@ generates:
   directory: src-gen
 exports:
 - language: Stratego-Sugar
-  directory: src-gen  
+  directory: trans
+  includes:
+  - "**/*.str"
+- language: Stratego-Sugar
+  directory: src-gen
+  includes:
+  - "**/*.str"
 - language: ATerm
   directory: src-gen/statix
+  includes:
+  - "**/*.stx"
 pardonedLanguages:
```

Comment:
This file is missing a `contributions` section to enable proper integration of the extension. See for example https://github.com/metaborg/nabl/blob/master/nabl2.lang/metaborg.yaml#L17-L19